translator/translate/logs/metrics_collected/prometheus/emfprocessor/ruleMetricUnit.go

```diff
@@ -0,0 +1,22 @@
+package emfprocessor
```

Comment:
Please add license headers in new source code files like this: https://github.com/aws/amazon-cloudwatch-agent/blob/a2da7b1e4ab0791d9608b1ec8a46ed8849a901d6/translator/translate/logs/metrics_collected/prometheus/prometheus.go#L1-L3

Reply:
Addressed in new commit.